storage/storage.go

```diff
@@ -5,6 +5,14 @@ import (
 	"github.com/trustwallet/watchmarket/pkg/watchmarket"
 )
 
+type SaveResult string
+const(
+	SaveResultSuccess                      SaveResult = "Success"
+	SaveResultStorageFailure               SaveResult = "StorageFailure"
+	SaveResultSkippedLowPriority           SaveResult = "SkippedLowPriority"
+	SaveResultSkippedLowPriorityOrOutdated SaveResult = "SkippedLowPriorityOrOutdated"
+)
```

Comment:
@EnoRage not sure if this is the Golang way to do this. Please let me know if there's a better way

Reply:
it's perfect